applications/HDF5Application/custom_io/hdf5_file_parallel.cpp

```diff
@@ -183,5 +184,434 @@ void FileParallel::ReadDataSetIndependent(const std::string& rPath,
     ReadDataSetMatrixImpl(rPath, rData, StartIndex, BlockSize, DataTransferMode::independent);
     KRATOS_CATCH("");
 }
+
+template <class T>
+void FileParallel::WriteDataSetVectorImpl(const std::string& rPath,
+                                          const Vector<T>& rData,
+                                          DataTransferMode Mode,
+                                          WriteInfo& rInfo)
+{
+    KRATOS_TRY;
+    BuiltinTimer timer;
+    // Expects a valid free path.
+    KRATOS_ERROR_IF(HasPath(rPath)) << "Path already exists: " << rPath << std::endl;
+
+    // Create any missing subpaths.
+    auto pos = rPath.find_last_of('/');
```

Comment:
win?

Reply:
yes, this is internal hdf5 path, not os path.